src/MongoDB/WriteConcern.c

```diff
@@ -100,6 +99,93 @@ PHP_METHOD(WriteConcern, __construct)
 }
 /* }}} */
 
+/* {{{ proto string|integer WriteConcern::getW()
+   Returns the WriteConcern "w" option */
+PHP_METHOD(WriteConcern, getW)
+{
+	php_phongo_writeconcern_t *intern;
+	const char *wtag;
+	(void)return_value_ptr; (void)return_value_used;
+
+	intern = (php_phongo_writeconcern_t *)zend_object_store_get_object(getThis() TSRMLS_CC);
+
+	if (zend_parse_parameters_none() == FAILURE) {
+		return;
+	}
+
+	wtag = mongoc_write_concern_get_wtag(intern->write_concern);
+
+	if (wtag) {
+		RETURN_STRING(wtag, 1);
+	}
+
+	if (mongoc_write_concern_get_wmajority(intern->write_concern)) {
+		RETURN_STRING(PHONGO_WRITE_CONCERN_W_MAJORITY, 1);
+	}
+
+	RETURN_LONG(intern->write_concern->w);
+}
+/* }}} */
+
+/* {{{ proto string|integer WriteConcern::getWtimeout()
+   Returns the WriteConcern "wtimeout" option */
+PHP_METHOD(WriteConcern, getWtimeout)
```

Comment:
http://docs.mongodb.org/master/core/write-concern/ referred to it simply as `wtimeout`, and our constructor argument is also just `$wtimeout`, so I wasn't inclined to capitalize it.

For what it's worth, I also think `getW()` looks really weird.